src/igvfd/schemas/tabular_file.json

```diff
@@ -187,6 +188,7 @@
                 "protein to protein interaction score": "Scores or metrics for interactions between proteins.",
                 "reporter elements": "Stores the raw format statistical activity analysis for elements associated with a reporter assay.",
                 "reporter experiment": "Stores the complete raw measurements from a reporter assay.",
+                "reporter experiment barcode": "Store the raw measurement on a barcode level from a reporter assay.",
```

Comment:
Did the MPRA group request specifically "barcode" and not "barcodes" in this enum?

Reply:
Yes.